src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -24,6 +24,7 @@ const FORMNAME_CLASSNAME_MAPPINGS = {
     [AUTHENTICATOR_KEY.OV]: 'mfa-verify',
     [AUTHENTICATOR_KEY.GOOGLE_AUTHENTICATOR]: 'mfa-verify',
     [AUTHENTICATOR_KEY.DUO]: 'mfa-verify-duo',
+    [AUTHENTICATOR_KEY.CUSTOM_OTP]: 'mfa-verify-custom-otp',
```

Comment:
I think this maps to V1, just make sure if we have this class on v1?

Reply:
Thanks for calling this out. Don't think this maps to a v1 class.